virt_lightning/shell.py

```diff
@@ -117,10 +117,16 @@ def up(configuration, virt_lightning_yaml_path, context):
 def ansible_inventory(configuration, context):
     hv = vl.LibvirtHypervisor(configuration)
 
+    ssh_cmd_template = (
+        "{name} ansible_host={ipv4} ansible_username={username} "
+        'ansible_ssh_common_args="-o UserKnownHostsFile=/dev/null '
```

Comment:
I see to redo this block with two ways:

1. Keep dictionary with options and convert to string
2. Add ssh configuration file and read file from ~/.config

Reply:
I assume you mean ~/.ssh/config, I would prefer to keep this directory intact.